resources/js/Pages/Admin/BookingRequests/CreateBookingRequestModal.vue

```diff
@@ -199,6 +203,9 @@ export default {
         this.createBookingRequestForm.reference.push(file);
       }
     },
+    setCreate(val) {
+      localStorage.create = val;
+    },
```

Comment:
```suggestion
    setLocalIsCreating(val) {
      localStorage.isCreatingBooking = val;
    },
```
the `setCreate` function name is a bit ambiguous, could you change it to be a bit more descriptive, as well as the `create` property. Included a suggestion for naming

Reply:
Agreed, and done :)